File: MigrationPy/pipelines/p1/code/p1/graph/Subgraph_1/Script_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.libs import typed_lit
from .config import *
from p1.udfs.UDFs import *

def Script_1(spark: SparkSession, in0: DataFrame) -> DataFrame:
    y = x
    out0 = in0.withColumn("x", lit(x))

    return out0

File: lalalala/pipelines/lineageRegression/code/src/main/scala/io/prophecy/pipelines/lineageregression/graph/Script_1.scala
package io.prophecy.pipelines.lineageregression.graph

import io.prophecy.libs._
import io.prophecy.pipelines.lineageregression.config.Context
import io.prophecy.pipelines.lineageregression.udfs.UDFs._
import io.prophecy.pipelines.lineageregression.udfs._
import io.prophecy.pipelines.lineageregression.udfs.PipelineInitCode._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Script_1 {
  def apply(context: Context, in0: DataFrame): DataFrame = {
    val spark = context.spark
    val Config = context.config
    import org.apache.spark.mllib.linalg.{Vector, Vectors}
    
    val out0 = in0.withColumn("prediction1", lit(Vectors.dense(1.0, 0.0, 3.0)))
    out0
  }

}
